src/proto/meta_protocol.rs

```diff
@@ -200,4 +212,52 @@ impl MetaProtocol for Client {
                 .transpose(),
         }
     }
+
+    async fn meta_arithmetic<K: AsRef<[u8]>>(
```

Comment:
Should we make this more convenient for the end-user by adding another "value" arg? And maybe providing an explicitly increment/decrement instead?

Reply:
We have been pushing down the route of just taking a set of flags so far but I do think that this might be a more compelling argument for explicitly taking params.

Could make this change to take a `delta` (with default of 1) and decrement as a boolean (default of `false`, as per the proto).  Other flags that should maybe be explicit params are q and opaque.

I have been thinking about this in the context of multi-ops as well where you would want to force the use of either an opaque or the k flag so that we can get away from this "trusting that responses are returned in order" issue that we have with that ASCII proto.  Having an opaque as a param makes this more straightforward.

Reply:
We talked about this IRL and decided that a good path would be exposing explicit `meta_{increment|decrement}` methods that handle adding the respective flags for each method. These can just call `meta_arithmetic` (private method) under the hood with the correct flags.